stringfuzz/generators/random_ast.py

```diff
@@ -160,9 +160,9 @@ def make_random_ast(num_vars, num_asserts, depth, max_terms, max_str_lit_length,
     asserts = [generate_assert(variables, depth) for i in range(num_asserts)]
 
     # add check-sat
-    asserts.append(CheckSatNode())
+    expressions = asserts + [CheckSatNode()]
```

Comment:
more readable? (just curious)

Reply:
Nah, it's just a pet peeve. It felt wrong to append a non-assert to a list called `asserts`, so I made another list to restore balance to the force.